frontend/src/features/documents/documentDetail/ComposedDocument.ts

```diff
@@ -0,0 +1,7 @@
+import { Api_Document } from 'models/api/Document';
+import { Mayan_DocumentMetadata } from 'models/api/DocumentManagement';
+
+export interface ComposedDocument {
```

Comment:
Hmm, not sure if we need this idea of composition in this case. it makes sense for properties since their are literally multiple different data sources at play, but here can't our backend hide that some data is in mayan and some is in our db? does our frontend need to know about this implementation detail?

Reply:
We could abstract it, but given we are not changing the model on the other concepts I believe is better if leave that as is. We could just called StoredDocumentModel or something like that to differentiate it from the pims one